src/nft/components/bag/BagRow.tsx

```diff
@@ -99,11 +101,11 @@ export const BagRow = ({ asset, usdPrice, removeAsset, showRemove, grayscale, is
   const showRemoveButton = Boolean(showRemove && cardHovered && !isMobile)
 
   const assetEthPrice = asset.updatedPriceInfo ? asset.updatedPriceInfo.ETHPrice : asset.priceInfo.ETHPrice
-  const assetEthPriceFormatted = formatWeiToDecimal(assetEthPrice)
-  const assetUSDPriceFormatted = ethNumberStandardFormatter(
-    usdPrice ? parseFloat(formatEther(assetEthPrice)) * usdPrice : usdPrice,
-    true
-  )
+  const assetEthPriceFormatted = formatEther({ input: assetEthPrice, type: NumberType.NFTToken })
+  const assetUSDPriceFormatted = formatNumberOrString({
+    input: usdPrice ? parseFloat(ethersFormatEther(assetEthPrice)) * usdPrice : usdPrice,
```

Comment:
Should this be overloaded so it doesn't need `ethersFormatEther` inside of it?

Reply:
no because in this case we need to get the ethers value then multiply by the price